webpack.config.commercial.js

```diff
@@ -16,6 +16,7 @@ config.entry = {
 
 module.exports = webpackMerge.smart(config, {
 	output: {
+		jsonpFunction: 'commercialJsonp',
```

Comment:
This 'magic' could probably do with a comment or a link to docs.
Should it also be removed once we upgrade to v5?

Reply:
Fair point. Will add. I think having a uniquely named entry point also helped.